tests/playwright/sanity/container.test.js

```diff
@@ -261,4 +261,32 @@ test.describe( 'Container tests', () => {
 			container: false,
 		} );
 	} );
+
+	test( 'Container handle should be centered', async ( { page }, testInfo ) => {
+		const wpAdmin = new WpAdminPage( page, testInfo );
+		await wpAdmin.setExperiments( {
+			container: true,
+		} );
+		try {
+			await wpAdmin.setLanguage( 'he_IL' );
```

Comment:
I believe at the end of the test we should return the language to English. 
What do you think?

Reply:
@hein-obox I believe this is happening in the `finally`?

Reply:
@louiswol94 

I am not sure if I get what you mean by `finally`?
I have added the language change at the of the test. I am not sure if you mean that or do you mean something else?

Reply:
@hein-obox 
as @louiswol94 said 
It does so in finally block

https://www.w3schools.com/jsref/jsref_try_catch.asp#:~:text=try%20%7B%0A%C2%A0%20tryCode%20%2D%20Code%20block%20to%20run%0A%7D%0Acatch(err)%20%7B%0A%C2%A0%20catchCode%20%2D%20Code%20block%20to%20handle%20errors%0A%7D%0Afinally%20%7B%0A%C2%A0%20finallyCode%20%2D%20Code%20block%20to%20be%20executed%20regardless%20of%20the%20try%20result%0A%7D